x-pack/plugins/fleet/server/plugin.ts

```diff
@@ -322,8 +322,8 @@ export class FleetPlugin
         },
       },
       agentService: {
-        getAgent,
-        listAgents,
+        getAgent: getAgentById,
+        listAgents: getAgentsByKuery,
```

Comment:
I am wondering if the fact that we do not use the same function name internally and externally will cause some communication issue in the future.

Reply:
Fair question. We can update these as well, I just held off while we're iterating.

Would you like me to update them now?

Reply:
Maybe this can be done in a follow up PR, as it's probably going to take some time to be reviewed as it concern multiple team